cfme/tests/ssui/test_ssui_dashboard.py

```diff
@@ -25,13 +27,100 @@
 ], scope="module")
 
 
-@pytest.mark.uncollectif(lambda: current_version() < '5.8')
 @pytest.mark.parametrize('context', [ViaSSUI])
-def test_ssui_dashboard(setup_provider, context, order_catalog_item_in_ops_ui):
-    """Tests various Primary and aggregate card values displayed on dashboard."""
+def test_total_services(setup_provider, context, order_catalog_item_in_ops_ui):
+    """Tests total services displayed on dashboard."""
+
+    appliance = get_or_create_current_appliance()
+    with appliance.context.use(context):
+        appliance.server.login()
+        dashboard = Dashboard(appliance)
+        dashboard.total_services()
```

Comment:
See my comments wrt this checks outside of tests. I thing it would be better to implement method like
```python
def num_of_rows(self):
    view.paginator.set_items_per_page("100 items")
    rows = view.paginator.items_amount
    return rows
```
and then in tests use smtg like ``assert dashboard.total_services() == dashboard.num_of_rows()``